chapter1/src/test/java/chapter1/MyFirstControllerTest.java

```diff
@@ -0,0 +1,25 @@
+package chapter1;
```

Comment:
maybe having the Controller test in the same package as the controller, `spring.training.controller`?